test/tests/smoke_test.rs

```diff
@@ -14,13 +14,25 @@ fn assert_output_contains(haystack: &str, needle: &str) {
 
 #[test]
 fn smoke_test() {
-    let rom = caliptra_builder::build_firmware_rom(&ROM_WITH_UART).unwrap();
-    let image = caliptra_builder::build_and_sign_image(
-        &FMC_WITH_UART,
-        &APP_WITH_UART,
-        ImageOptions::default(),
-    )
-    .unwrap();
+    #[cfg(not(feature = "fpga_realtime"))]
+    let (rom, fmc, app) = { (ROM_WITH_UART, FMC_WITH_UART, APP_WITH_UART) };
+
+    #[cfg(feature = "fpga_realtime")]
```

Comment:
This isn't the only test folks want to run on the FPGA. Maybe we should do this globally in caliptra_builder?

https://github.com/chipsalliance/caliptra-sw/blob/main/builder/src/lib.rs#L118

```
if cfg!(feature = "fpga_realtime") {
  features_csv.push_str("fpga_realtime");
}
```

Reply:
Will do.